Sources/BridgeArchiver/BridgeArchiver.swift

```diff
@@ -150,11 +148,10 @@ public class BridgeArchiver {
         }
         let encryptedURL = url ?? archiveURL.appendingPathExtension("encrypted")
         let data = try Data(contentsOf: archiveURL)
-        #if canImport(CMSSupport)
-        let encryptedData = try CMSSupport.cmsEncrypt(data, identityPath: pemPath)
-        #else
-        let encryptedData = data
-        #endif
+        let (encryptedData, encrypted) = try CMSEncryption.cmsEncrypt(data, identityPath: pemPath)
```

Comment:
If cmsEncrypt throws, won't this throw before ever returning "false" for encrypted? Or does it throw for some other condition and could still return false?

Reply:
Correct, it *could* throw on iOS for some other reason (ex. corrupted PEM file) but that's a *different* reason from sending encrypted data to S3 is intentionally disabled for the Mac b/c we don't support it.